data_structures/src/chain.rs

```diff
@@ -4006,7 +4009,7 @@ mod tests {
     #[test]
     fn test_transaction_hashable_trait() {
         let transaction = transaction_example();
-        let expected = "7b4001b4a43b3e3dccec642791031d8094ea52164d89c2a4732d0be79ed1af83";
+        let expected = "9221b651aaef0f6d75a175958825775ed0add54102a431a27e485052478e0f9b";
```

Comment:
The hash of the `transaction_example()` changes because the `RADType` changes from 0 to 1.